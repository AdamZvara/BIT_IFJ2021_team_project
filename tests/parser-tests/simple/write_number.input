-- cislo 1.0 by to melo podle tl prekladace vytisknout takto 0x1p+0
-- ale interpret co jsme dostali od skoly tak z toho float cisla udela 1
-- ale nemyslim si ze je chyba na nasi strane

require "ifj21"

write(1.0)
write(" ", 2.25)
