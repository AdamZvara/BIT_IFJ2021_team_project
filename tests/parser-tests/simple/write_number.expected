1 0x1.2p+1